.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -25,6 +25,7 @@ body:
       label: Logs
       description: |
         Please paste your client logs (logcat) here, *NOT* server logs (in most cases).
+        In order to capture useful information in the logs, "debug options" should be enabled in the app settings.
```

Comment:
The debug option in the advanced settings doesn't actually enable debugging features. It only adds media information to the FullDetailsActivity. Nothing changes with actual logs.